projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -1053,4 +1054,32 @@ class="widefat filter-count"
 		</div>
 		<?php
 	}
+
+	/**
+	 * Returns the taxonomies for search widget taxonomy dropdown.
+	 */
+	protected function get_allowed_taxonomies_for_widget_filters() {
+		return array_filter(
+			get_taxonomies( array( 'public' => true ), 'objects' ),
+			function ( $taxonomy ) {
+				return in_array(
+					$taxonomy->name,
+					/**
+					 * Filters the taxonomies that shows in filter drop downs of the search widget.
+					 *
+					 * @since  $$next-version$$
```

Comment:
Do we need to change this or is it automatically populated?

Reply:
This is a place holder for the next version, which will be replaced on release 😄 

The PR that brought this should be this one: https://github.com/Automattic/jetpack/pull/20386

Reply:
Ah nice! TIL.